configs/textrecog/seg/convert.py

```diff
@@ -0,0 +1,48 @@
+import torch
```

Comment:
The file is a tool, so it could be put at `tools/deployment/pytorch2torchscript.py`